mopro-core/Cargo.toml

```diff
@@ -11,9 +11,9 @@ edition = "2021"
 wasmer = { git = "https://github.com/oskarth/wasmer.git", rev = "09c7070" }
 
 [features]
-default = ["wasmer/dylib"]
-dylib = []                                        # NOTE: can probably remove this if we use env config instead
-gpu-benchmarks = ["jemalloc-ctl", "jemallocator"]
+default = []
```

Comment:
You need to rebase on master, this is not the latest version

Reply:
sorry I'll reset and rebase again.